### PR TITLE
Reinstall reorganization swope

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: apis/clusterinstance.json.adoc [hunk 1]
```diff
@@ -450,7 +450,7 @@ __optional__|Specify if automatic introspection runs during registration of the
 |string
 |*nodeLabels* +
 __optional__|Specify custom node labels for your nodes in your managed clusters.
-These are additional labels that are not used by any {product-title} components, only by the user.
```
Comment (swopebe) [inline apis/clusterinstance.json.adoc:453]: This is unrelated, only a change to ensure we caught all the conrefs--I found this and it will need to change in all versions. This file does not need a review.

File: troubleshooting/troubleshooting_intro.adoc [hunk 2]
```diff
@@ -16,8 +16,6 @@ To view the main documentation for the installing tasks, see link:../install/ins
 
 - xref:../troubleshooting/trouble_install_status.adoc#troubleshooting-stuck-pending[Troubleshooting installation status stuck in installing or pending]
 
-- xref:../troubleshooting/trouble_reinstall.adoc#troubleshooting-reinstallation-failure[Troubleshooting reinstallation failure]
```
Comment (swopebe) [inline troubleshooting/troubleshooting_intro.adoc:19]: removing this bc file does not exist.

File: troubleshooting/main.adoc [hunk 3]
```diff
@@ -6,7 +6,6 @@ include::troubleshooting_intro.adoc[leveloffset=+1]
 include::must_gather.adoc[leveloffset=+2]
 include::trouble_hosted_cluster.adoc[leveloffset=+3]
 include::trouble_install_status.adoc[leveloffset=+2]
-include::trouble_reinstall.adoc[leveloffset=+2]
```
Comment (swopebe) [inline troubleshooting/main.adoc:9]: file is removed

File: install/reinstall.adoc [hunk 4]
```diff
@@ -1,28 +1,25 @@
-[#troubleshooting-reinstallation-failure]
-= Troubleshooting reinstallation failure
+[#cleanup-reinstall]
```
Comment (swopebe) [inline install/reinstall.adoc:1]: New file for entire review, the script is required.

File: clusters/install_upgrade/uninstall.adoc [hunk 5]
```diff
@@ -20,7 +20,7 @@ Cannot delete MultiClusterEngine resource because ManagedCluster resource(s) exi
 ----
 
 +
-For more information about detaching clusters, see the _Removing a cluster from management_ section by selecting the information for your provider in xref:../cluster_lifecycle/create_intro.adoc#create-intro[Cluster creation introduction].
+For more information about detaching clusters, see the _Removing a cluster from management_ section by selecting the information for your provider in xref:../cluster_lifecycle/create_intro.adoc#create-intro[Creating clusters].
```
Comment (swopebe) [inline clusters/install_upgrade/uninstall.adoc:24]: filename was incorrect

File: governance/manage_operator.adoc [hunk 6]
```diff
@@ -11,7 +11,7 @@ Complete the following steps to validate access to the operator images:
 - **Deprecated:** `gatekeeper-operator-product`
 - `compliance-operator`
 +
-. See link:../install/install_disconnected.adoc#disconnect-configure-icsp[Configure image content source policies] to validate that the sources are available. The image content source policies must exist on each of the disconnected managed clusters and can be deployed using a policy to simplify the process. See the following table of image source locations: 
+. See link:../install/install_disconnected.adoc#disconnect-config-icsp[Configure image content source policies] to validate that the sources are available. The image content source policies must exist on each of the disconnected managed clusters and can be deployed using a policy to simplify the process. See the following table of image source locations:
```
Comment (swopebe) [inline governance/manage_operator.adoc:14]: This is only to shorten the URL, this file does not need review.

File: install/install_disconnected.adoc [hunk 7]
```diff
@@ -6,9 +6,9 @@ You might need to install {acm} on disconnected {ocp} clusters. To install on a
 *Required access:* Cluster administrator
 
 * <<disconnect-prerequisites,Prerequisites>>
-* <<disconnect-confirm-ocp-installation,Confirm your {ocp-short} installation>>
-* <<disconnect-configure-olm,Configure {olm}>> 
-* <<disconnect-configure-icsp,Configure image content source policies>>
+* <<disconnect-confirm-ocp-install,Confirming your {ocp-short} installation>>
+* <<disconnect-config-olm,Configuring {olm}>> 
+* <<disconnect-config-icsp,Configuring image content source policies>>
 * <<disconnect-install-op-and-hub,Install the {acm} operator and hub>>
```
Comment (swopebe) [inline install/install_disconnected.adoc:12]: header changes only

File: install/install_connected.adoc [hunk 8]
```diff
@@ -17,6 +17,8 @@ You must have a supported version of {ocp-short} to install {acm-short}.
 
 - If you plan to import Kubernetes clusters that are not {ocp-short} or {acm-short} clusters, you need to configure an image pull secret. 
 
+- If you previously installed {acm-short}, then uninstalled, you need to follow the clean up instructions to remove artifacts. See xref:../install/reinstall.adoc##cleanup-reinstall[Cleaning up artifacts before reinstalling] and follow the procedure.
+
```
Comment (swopebe) [inline install/install_connected.adoc:21]: Review line 20

File: install/install_connected.adoc [hunk 9]
```diff
@@ -254,8 +256,6 @@ error: unable to recognize "./mch.yaml": no matches for kind "MultiClusterHub" i
 oc get mch -o yaml
 ----
 
-If you are reinstalling {acm-short} and the pods do not start, see link:../troubleshooting/trouble_reinstall.adoc#troubleshooting-reinstallation-failure[Troubleshooting reinstallation failure] for steps to work around this problem.
-
```
Comment (swopebe) [inline install/install_connected.adoc:258]: removing the t-shooting file that the new file replaces. No need to review.

File: install/install_disconnected.adoc [hunk 10]
```diff
@@ -24,7 +24,9 @@ You must meet the following requirements before you install {acm}:
 
 * Review xref:../install/cluster_size.adoc#sizing-your-cluster[Sizing your cluster] to learn about setting up capacity for your hub cluster.
 
-[#disconnect-confirm-ocp-installation]
+- If you previously installed {acm-short}, then uninstalled, you need to follow the clean up instructions to remove artifacts. See xref:../install/reinstall.adoc##cleanup-reinstall[Cleaning up artifacts before reinstalling] and follow the procedure.
+
```
Comment (swopebe) [inline install/install_disconnected.adoc:28]: same description as above.

File: install/install_disconnected.adoc [hunk 11]
```diff
@@ -24,7 +24,9 @@ You must meet the following requirements before you install {acm}:
...
+[#disconnect-confirm-ocp-install]
```
Comment (swopebe) [inline install/install_disconnected.adoc:29]: only header, no content changes

File: install/reinstall.adoc [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+== Cleaning up artifacts before reinstalling
+
+Before you reinstall {acm} on a cluster or a previous version was installed and then deleted, you might need to remove artifacts.
+//how do they know if they need to remove artifacts?
```
Comment (swopebe) [inline install/reinstall.adoc]: Before you reinstall {acm} on a cluster or a previous version was installed and then deleted...

This is confusing to me, are we trying to say:

_Before you reinstall {acm} on a cluster where a previous version was installed and then deleted..._

File: install/reinstall.adoc [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+**Required access:** Cluster administrator. 
+
+**{ocp-short} Dedicated environment required access:** You must have `cluster-admin` permissions.
+
+[#prereqs-reinstall]
+== Prerequisites
+
+- You are using the same cluster access as the install connnected or disconnected guides require.
+//need some clarity on this line and are there any other prereqs?
```
Comment (swopebe) [inline install/reinstall.adoc]: Since we give them the cluster access required, do we need this or are we trying to say something else?

File: install/uninstall.adoc [hunk 14]
```diff
@@ -60,6 +59,7 @@ When you delete the resource, the pods in the `open-cluster-management-observabi
 
 [#removing-a-multiclusterhub-instance-by-using-commands]
 == Removing _MultiClusterHub_ resources by using commands
+//this whole section needs removed, correct?
```
Comment (swopebe) [inline install/uninstall.adoc:62]: We need to remove everything that applies here

File: install/uninstall.adoc [hunk 15]
```diff
@@ -153,10 +153,6 @@ When you use the {ocp} console to uninstall, you remove the `MultiClusterHub` re
 
 .. Select *Delete MultiClusterHub*.
 
-. Run the clean-up script according to the procedure in xref:../install/uninstall.adoc#removing-a-multiclusterhub-instance-by-using-commands[Removing a MultiClusterHub instance by using commands].
-+
-*Note:* If you plan to reinstall the same {acm-short} version, you can skip the rest of the steps in this procedure and reinstall the custom resource.
-
 . Navigate to *Installed Operators*.
```
Comment (swopebe) [inline install/uninstall.adoc:129]: This now gets removed, however, we need to ensure the section is still correct.

File: install/reinstall.adoc (hunk 12 above)
Comment (cameronmwall) [reply install/reinstall.adoc]: I think we can be more emphatic here and just say you will need to remove. It is best practice for them to perform the cleanup. I think your wording is good

File: install/reinstall.adoc (hunk 13 above)
Comment (cameronmwall) [reply install/reinstall.adoc]: no, I think we are repeating that they need cluster administrator access. We can remove the redundancy

File: install/uninstall.adoc (hunk 14 above)
Comment (cameronmwall) [reply install/uninstall.adoc:62]: I think we can just remove step 5, as that is the info that is now in our reinstall doc. The other info is still relevant for a normal uninstall

Comment (swopebe) [reply install/uninstall.adoc:62]: Done

File: install/reinstall.adoc (hunk 13 above)
Comment (swopebe) [reply install/reinstall.adoc]: Done

**Comment by swopebe** [comment]
Merging to get the file removed and sections removed for a final review and peer review.
